config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  devise_for :users
+  root 'posts#index'
+  resources :posts
```

Comment:
- [ ] The posts routes creates the application routes using `resources :posts, only: \[:new, :create, :index\]`